tests/valid_login.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { test, expect } from '@playwright/test';
+
+test('login with valid credential', async ({ page }) => {
+    await page.goto('https://practicesoftwaretesting.com/auth/login');
```

Comment:
When you add baseURL in config, you can remove it from here, and leave only path to the page: 
```suggestion
    await page.goto('/auth/login');
```